### ISSUE TITLE
GUI changes

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#68]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Add menu bar to gui
* Add license popup to display LGPL license of QT, MIT license of our code
* Remove license popup before app launch
* Add small logo with text to app
* Use Qt Group box to group elements which belong together. Give the boxes representative titles
* Remove "Send Message"  button
* Change Window width to 1200
* Optional: Make CAN Bus selection collapsable

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
